### ISSUE TITLE
Consolidate some legacy test constructs

### ISSUE THREAD

**Issue by dmnks** [rpm-software-management/rpm#2773]
* [ ] Merge `RPMDB_INIT` into `RPMTEST_SETUP`
* [ ] Rename `RPMTEST_SETUP` to `RPMTEST_INIT`
* [ ] Add `RPMTEST_SETUP` as a **wrapper** for `AT_SETUP` (for consistency)
* [x] Drop `run()`
* [ ] (stretch goal) Define wrappers for the RPM binaries (`rpm`, `rpmbuild`, `rpmsign`, ...) which would be used instead of `runroot*()` (optimization: don't run a Bubblewrap container if `$RPMTEST` is `/`)
* [ ] ...and finally eliminate the need for explicit RPMTEST_INIT at all

Thanks @pmatilai for the ideas! :smile:

**Comment by dmnks** [comment]
>     * Perhaps `RPMTEST_SETUP` isn't very descriptive, choose a better name? Its purpose is to initialize a writable tree and mount it at `$RPMTEST`

Hmm, actually  this is probably a good name after all :smile: It literally says "setup RPMTEST (directory)"...

**Comment by pmatilai** [comment]
The problem with RPMTEST_SETUP name is that it relates AT_SETUP when it doesn't. Especially as we have RPMTEST_CLEANUP which relates to AT_CLEANUP. Maybe RPMTEST_INIT?

**Comment by dmnks** [comment]
Yup, good point. It *could* be made into an `AT_SETUP` wrapper and just take some kind of argument that says "I need write access to `/`" but that would require a *lot* of (even if automated) search & replace across all the test files...

In any case, `RPMTEST_INIT` works for me.

**Comment by pmatilai** [comment]
After it's renamed, it may not be a bad idea to actually add RPMTEST_SETUP that does, for now, nothing more than wrap AT_SETUP. Just to make the whole thing more consistent.

**Comment by dmnks** [comment]
OK, I've updated the description accordingly, thanks!

**Comment by pmatilai** [comment]
Yay, we got to drop run() :smile:

**Comment by pmatilai** [comment]
Seems this has turned into halfway rewrite by now 😂 but it'd seem strange not to include https://github.com/rpm-software-management/rpm/pull/3542 in this as well...

**Comment by pmatilai** [comment]
I think we'll consider this step done now. There are other legacy constructs to deal with but those are better dealt with on in their own tickets.
